matzip-app-external-api/src/test/java/com/woowacourse/acceptance/RestaurantAcceptanceTest.java

```diff
@@ -127,4 +143,13 @@ public class RestaurantAcceptanceTest extends AcceptanceTest {
                 () -> assertThat(response.as(RestaurantTitlesResponse.class).isHasNext()).isTrue()
         );
     }
+
+    private void 캠퍼스_식당_추천_검색어_자동완성에_성공한다(final ExtractableResponse<Response> response, int size, String... names) {
+        assertAll(
+                () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value()),
+                () -> assertThat(response.as(RestaurantSearchesResponse.class).getRestaurants()).extracting("name")
+                        .containsExactly(names),
+                () -> assertThat(response.as(RestaurantSearchesResponse.class).getRestaurants()).hasSize(size)
```

Comment:
```suggestion
                () -> assertThat(response.as(RestaurantSearchesResponse.class).getRestaurants()).hasSize(size)
                .extracting("name")
                .containsExactly(names)
```

이렇게 좀 더 간결하게 표현해도 좋을 것 같아요!

Reply:
좋은 의견 감사합니다:)
저는 그렇게 작성해도 좋지만 '사이즈가 몇 인가' 와 'names를 포함하고 있는가' 는 다른 테스트라고 생각해서 분리해두었습니다.

테스트라는 것은 물론 코드 검증 역할도 있지만, 문서의 역할도 한다고 생각해서
성능 상 문제가 발생하는 부분이 아니라면 알아보기 쉬운 코드를 선택해서 작성하는 편인데 바꾸는 것이 좋을까요??

Reply:
`캠퍼스의_식당_중_입력받은_접두사로_시작하는_이름의_식당을_좋아요_기준_내림차순으로_조회한다` 테스트 코드의 방식과 통일성이 안맞아보여서 남겼습니다.

테스트하고자 하는 부분이 정확한 응답 바디를 주고 있는지 확인하는 것이기 때문에 한줄로 줄여도 알아보는데 어려움은 없을 것 같아요.

Reply:
말씀해주셔서 다시 봤는데 그 부분은 제가 3개가 잘 오는지 개인적으로 검증하려고 코드를 써두고 지우지 못했네요!
추가로 containsAnyOf 로 값을 검증하고 있었는데 3개가 순서에 맞게 제대로 조회되어왔는 지 검사하는 것이 목적이기 때문에 containsExactly로 변경하였습니다:)